pkg/apis/istio/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,5 +1,21 @@
 // +build !ignore_autogenerated
 
+/*
+Copyright The Kubernetes Authors.
```

Comment:
Why do we see this all of the sudden?

Reply:
hmm.. Good question. Some lib update I guess